.travis.yml

```diff
@@ -31,44 +38,85 @@ jobs:
   include:
     - php: nightly
       env: COMPOSER_FLAGS="--ignore-platform-reqs"
+
     - stage: test with coverage
       php: 7.1
       before_install:
         - stty cols 130
         - echo 'opcache.enable=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
         - echo 'opcache.enable_cli=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
         - travis_retry composer self-update
-      script: vendor/bin/phpunit -c phpunit.xml.dist --colors --columns 121 --verbose --coverage-text --coverage-clover=coverage.clover
+      script: vendor/bin/phpunit -c phpunit.xml.dist --colors --columns 117 --verbose --coverage-text --coverage-clover=coverage.clover
       after_success:
-        - wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover coverage.clover
-        - bash <(curl -s https://codecov.io/bash) -f coverage.clover
+        - bash <(curl -s https://codecov.io/bash) -f coverage.clover -F phpunit
```

Comment:
Ah, I think this has come up before. I don't think it's worth flagging the PHPUnit tests with the "phpunit" flag, since they really won't ever have cross-coverage with the "full"/"lite"/"standard" flags, and just pollutes the codecov UI a little bit.

Reply:
fixed